help/moderation/appeals.md

```diff
@@ -0,0 +1,52 @@
+- [Can all decisions be appealed?](#final)
+
+<span id="what-is"></span>
+## What is a moderation appeal?
+
+The appeal is a request sent to arXiv staff. In general, appeals are for additional scrutiny of moderation [decisions before](/help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 
+
+Your appeal message must include all relevant details, such as paper/submit IDs, former correspondence, etc. It must also include your rationale for why the moderators should reconsider the work. This should include a detailed description of the research content of your article, and how the content of your paper directly applies to that category (see the [category taxonomy](/category_taxonomy) for specific category descriptions). For declined submissions, it may be desirable for you to include a pdf copy of your work when appealing, as they are not retained in perpetuity. 
+
+You are not communicating directly with the moderators, and it is not appropriate to attempt to contact them directly. The moderators are encouraged to not reply to such messages. 
+
+<span id="what-happens"></span>
+## What happens when I appeal?
+
+You will receive a response from arXiv staff when your appeal has been sent to our appellate moderators for consideration. This response will include a ticket identifier that we will use to keep track of your appeal. 
+
+The appeals process begins after acknowledgment of your appeal. If your submission was recently declined, arXiv staff will restore your submission to [on hold](/help/submit_status#on_hold) status, if possible.  
+
+The appellate moderator will be sent any correspondence related to the work, and will begin to consider the content of your appeal as well as the original moderation decision. Please be patient, as moderation appeals may be complicated and take some time. Appeals are handled in addition to the moderators’ daily duties, professional work, etc. Most appeals are resolved within a two week period, but some may take longer. If you have not heard from arXiv within two weeks of receiving the notification that your appeal has been received, please feel free to reach out to us for a status update through the appropriate contact channel.
+
+When a decision is made by the appellate moderators, you will receive a follow-up message from arXiv staff with the results of your appeal. 
+
+<span id="accepted"></span>
+## What happens if my appeal is accepted? 
+
+If your appeal is accepted, the message you receive from arXiv staff will have instructions for resubmission (if needed), or will provide you with the [status](/help/submit_status) of your submission. In some instances it may be required that you make some changes in order to accommodate the appellate moderators' decision. 
+
+*Note* that for submissions that have been reclassified, they will not receive a "new" announcement within the changed category. 
```

Comment:
```suggestion
*Note:* Submissions that have been reclassified will not receive an announcement within the new category. 
```

Reply:
Looking at this with fresh eyes, I don't want to use "submissions" here, because this makes it sound like we're not going to announce moved submit-ids when we're talking about announced articles. I'm going to accept this but change submission to public articles.